### DISCUSSION TITLE
Trouble connecting to xdrip after many months of use

### DISCUSSION THREAD

**Discussion by 1303thomas** [NightscoutFoundation/xDrip#2452] [Q&A]
Having a problem connecting via xdrip. See thread for all screenshots. Was working until last night, clean reinstall do not work, most recent build does not work, latest stable does not work, beta does not work, older versions do not work. Rebooting the tomato device twice did not work. Clearing general cache and Bluetooth cache did not work. 

 Manual scan does work. 

US Libre 14 day sensor, miaomiao 2 bt bridge, oop2 most recent build installed, Android 13, Samsung 22S Ultra. Worked until last night, only change was new sensor, old sensor was properly stopped in app.

**Comment by 1303thomas** [comment]
![Screenshot_20221111_223622_xDrip+](https://user-images.githubusercontent.com/87401427/201456279-95a01dbe-6bd0-4190-802f-4eed85c7e020.jpg)
![Screenshot_20221111_224359_xDrip+](https://user-images.githubusercontent.com/87401427/201456280-6382f9b3-a986-4083-b205-856fd4531589.jpg)
![Screenshot_20221111_224405_xDrip+](https://user-images.githubusercontent.com/87401427/201456281-17a011bd-64bd-4542-ae4a-35a8f4349123.jpg)

**Reply by Navid200** [reply]
@psonnera
